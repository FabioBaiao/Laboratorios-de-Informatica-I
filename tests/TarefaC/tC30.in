aCa
aca
aca
aee
bcd
aac
aac
0 6 S
AAAASESSDSALEEASSEADSAAL
