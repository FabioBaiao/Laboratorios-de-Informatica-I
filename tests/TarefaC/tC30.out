1 6
FIM 22
